src/Sulu/Bundle/CategoryBundle/Category/KeyWordManager.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\CategoryBundle\Category;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Sulu\Bundle\CategoryBundle\Entity\Category;
+use Sulu\Bundle\CategoryBundle\Entity\KeyWord;
+
+/**
+ * Manages keyword for categories.
+ */
+class KeyWordManager implements KeyWordManagerInterface
+{
+    /**
+     * @var KeyWordRepositoryInterface
+     */
+    private $repository;
```

Comment:
I would prefer to include the entity name to this variable. If we later inject more repositories this might lead to confusing situations.